docs/workspaces.md

```diff
@@ -601,6 +597,26 @@ the storage solution that you are using.
 * `ReadWriteMany` is the least commonly available Access Mode. If you use this access mode and these volumes are available
   to all Nodes within your cluster, you may want to disable the Affinity Assistant.
 
+### Persistent Volume Availability Zone
+
+`Persistent Volumes` are usually "zonal" (i.e. they live within a single Availability Zone and cannot be accessed from a `pod` living in another Availability Zone). When using a workspace backed by a `PersistentVolumeClaim` (typically only available within a Data Center), the `TaskRun`
+`pods` can be scheduled to any Availability Zone in a regional cluster. This leads to two kinds of potential Availability Zone scheduling conflict.
+To avoid such conflict in `PipelineRuns`, Tekton provides [Affinity Assistants](affinityassistants.md) that schedule all `PipelineTaskRun` Pods or all `PipelineTaskRun` sharing a `PersistentVolumeClaim` to the same Node depending on the `coschedule` mode. 
+
+#### `PipelineTaskRuns` sharing the same `PersistentVolumeClaim` are scheduled to different Availability Zones.
```

Comment:
Can this docs section focus more on the features Tekton provides? It sort of works backwards from the problems you might run into to the features we built to solve them, but isn't super clear about how you would know if this is something you should enable. these features are most useful for:
- people using zonal clusters e.g. GKE
- people using pvcs w/ ReadWriteOnce access modes
- people binding multiple PVCs to a single pipeline task
And then explain what the features do or link to the existing docs on them.

Reply:
Thanks. It turns out that I emphasis too much on explaining the conflict so the focus is a bit off.

I have reframe the whole section, focusing on the feature that is provided by Tekton. PTAL

Reply:
This looks a lot better!